src/electionguard_gui/web/components/admin/admin-home-component.js

```diff
@@ -6,6 +6,21 @@ export default {
     KeyCeremonyList,
     ElectionsList,
   },
+  data() {
+    return {
+      loading: true,
+      keyCeremonies: [],
+    };
+  },
+  async mounted() {
+    const result = await eel.get_key_ceremonies()();
```

Comment:
In refactoring the guardian home page to pull all data so the watching operation could happen there, the admin home now also needs to pull data and push it down to the key ceremony component.  A nice side-effect is the admin never need to watch for updated key ceremonies, so now it doesn't.